vhdbuilder/packer/vhd-image-builder-mariner-arm64.json

```diff
@@ -509,6 +509,21 @@
       "source": "parts/linux/cloud-init/artifacts/pam-d-su",
       "destination": "/home/packer/pam-d-su"
     },
+    {
+      "type": "file",
+      "source": "parts/linux/cloud-init/artifacts/mariner/mariner-package-update.sh",
+      "destination": "/home/packer/mariner-package-update.sh"
+    },
+    {
+      "type": "file",
+      "source": "parts/linux/cloud-init/artifacts/mariner/package-update.service",
+      "destination": "/home/packer/snapshot-update.service"
+    },
+    {
+      "type": "file",
+      "source": "parts/linux/cloud-init/artifacts/mariner/package-update.timer",
+      "destination": "/home/packer/snapshot-update.timer"
```

Comment:
if we're putting this on the VHD directly then we _could_ just be able to enable the timer at build time so it automatically starts on subsequent boots (e.g. CSE), though probably fine to leave as is with `ensureSnapshotUpdate` in cse_main.sh

Reply:
Since we also leverage nodecustomdata, I stay with ensureSnapshotUpdate approach.